.github/workflows/development.yml

```diff
@@ -0,0 +1,147 @@
+      run: echo acr=$(az acr show -n ${{ env.cr }} --query loginServer | tr -d '"') >> $GITHUB_ENV
+
+    - name: Defaults ✨
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          # Basic
+          az configure --defaults location=${{ vars.REGION }}
+          az configure --defaults group=rg-${{ env.product }}-${{ env.environment }}-001
+
+    - name: ACR 🔐
+      uses: azure/docker-login@v1
+      with:
+        login-server: ${{ env.acr }}
+        username: ${{ secrets.ACR_USERNAME }}
+        password: ${{ secrets.ACR_PASSWORD }}
+    
+    - name: Artifacts 🗃️
+      run: |
+        docker build . \
+        -t ${{ env.acr }}/${{ env.name }}:${{ github.sha }} \
+        -t ${{ env.acr }}/${{ env.name }}:${{ env.from }}
+        docker push ${{ env.acr }}/${{ env.name }}:${{ github.sha }}
+        docker push ${{ env.acr }}/${{ env.name }}:${{ env.from }}
+
+    - name: Revisions 🔀
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          az containerapp update \
+          --name ca-mdm-${{ env.product }}-${{ env.environment }}-${{ vars.REGION }}-001 \
+          --image ${{ env.acr }}/${{ env.name }}:${{ env.from }} \
+          --set-env-vars \
+          "PORT=${{ secrets.PORT }}" \
+          "NODE_ENV=${{ secrets.NODE_ENV }}" \
+          "TZ=${{ secrets.TZ }}" \
+          "SWAGGER_USER=${{ secrets.SWAGGER_USER }}" \
+          "SWAGGER_PASSWORD=${{ secrets.SWAGGER_PASSWORD }}" \
+          "DATABASE_MDM_HOST=${{ secrets.DATABASE_MDM_HOST }}" \
+          "DATABASE_MDM_PORT=${{ secrets.DATABASE_MDM_PORT }}" \
+          "DATABASE_MDM_USERNAME=${{ secrets.DATABASE_MDM_USERNAME }}" \
+          "DATABASE_MDM_PASSWORD=${{ secrets.DATABASE_MDM_PASSWORD }}" \
+          "DATABASE_MDM_NAME=${{ secrets.DATABASE_MDM_NAME }}" \
+          "DATABASE_NUMBER_GENERATOR_HOST=${{ secrets.DATABASE_NUMBER_GENERATOR_HOST }}" \
+          "DATABASE_NUMBER_GENERATOR_PORT=${{ secrets.DATABASE_NUMBER_GENERATOR_PORT }}" \
+          "DATABASE_NUMBER_GENERATOR_USERNAME=${{ secrets.DATABASE_NUMBER_GENERATOR_USERNAME }}" \
+          "DATABASE_NUMBER_GENERATOR_PASSWORD=${{ secrets.DATABASE_NUMBER_GENERATOR_PASSWORD }}" \
+          "DATABASE_NUMBER_GENERATOR_NAME=${{ secrets.DATABASE_NUMBER_GENERATOR_NAME }}" \
+          "DATABASE_CEDAR_HOST=${{ secrets.DATABASE_CEDAR_HOST }}" \
+          "DATABASE_CEDAR_PORT=${{ secrets.DATABASE_CEDAR_PORT }}" \
+          "DATABASE_CEDAR_USERNAME=${{ secrets.DATABASE_CEDAR_USERNAME }}" \
+          "DATABASE_CEDAR_PASSWORD=${{ secrets.DATABASE_CEDAR_PASSWORD }}" \
+          "DATABASE_CEDAR_NAME=${{ secrets.DATABASE_CEDAR_NAME }}" \
+          "DATABASE_LOCAL_CEDAR_HOST=${{ secrets.DATABASE_LOCAL_CEDAR_HOST }}" \
+          "DATABASE_LOCAL_CEDAR_PORT=${{ secrets.DATABASE_LOCAL_CEDAR_PORT }}" \
+          "DATABASE_LOCAL_CEDAR_USERNAME=${{ secrets.DATABASE_LOCAL_CEDAR_USERNAME }}" \
+          "DATABASE_LOCAL_CEDAR_PASSWORD=${{ secrets.DATABASE_LOCAL_CEDAR_PASSWORD }}" \
```

Comment:
The env with `LOCAL`are not needed. We used them locally for testing purposes

Reply:
Good spot, have removed.